content/hi/docs/setup/best-practices/cluster-large.md

```diff
@@ -0,0 +1,88 @@
+---
+title: बड़े क्लस्टर्ज़ के लिए विचार
+weight: 20
+---
+
+एक क्लस्टर {{< glossary_tooltip text="नोड्स" term_id="node" >}} (भौतिक या वर्चुअल मशीन) का एक सेट है जो कुबेरनेट्स एजेंटों को चला रहा है, जिसे {{< glossary_tooltip text="कण्ट्रोल प्लेन" term_id="control-plane" >}} द्वारा प्रबंधित किया जाता है|
+कुबेरनेट्स {{< param "version" >}} 5000 नोड्स तक क्लस्टर का समर्थन करता है| अधिक विशेष रूप से, कुबेरनेट्स को ऐसे कॉन्फ़िगरेशन को समायोजित करने के लिए डिज़ाइन किया गया है जो निम्नलिखित *सभी* मानदंडों को पूरा करते हैं: 
+
+* प्रति नोड 110 पॉड से अधिक नहीं
+* 5000 से अधिक नोड्स नहीं
+* कुल पॉड्स 150000 से अधिक नहीं
+* 300000 से अधिक कुल कंटेनर नहीं 
+
+आप नोड्स जोड़कर या हटाकर अपने क्लस्टर को स्केल कर सकते हैं। आपके ऐसा करने का तरीका इस बात पर निर्भर करता है कि आपका क्लस्टर कैसे डिप्लॉय किया गया है|
+
+## क्लाउड प्रदाता संसाधन कोटा
+
+कई नोड्स के साथ क्लस्टर बनाते समय, क्लाउड प्रदाता कोटा मुद्दों में भाग लेने से बचने के लिए, इस पर विचार करें:
+
+* क्लाउड संसाधनों के लिए कोटा बढ़ाने का अनुरोध करना, जैसे: 
+    * कंप्यूटर उदाहरण
+    * CPU
+    * स्टोरेज वॉल्यूम
+    * उपयोग में आने वाले IP पते
+    * पैकेट फ़िल्टरिंग नियम सेट
+    * लोड बैलेंसरों की संख्या
+    * नेटवर्क सबनेट
+    * लॉग स्ट्रीम
+* बैचों के बीच एक ठहराव के साथ, बैचों में नए नोड्स लाने के लिए क्लस्टर स्केलिंग क्रियाओं को गेट करना, क्योंकि कुछ क्लाउड प्रदाता नए इंस्टेंस के निर्माण को सीमित करते हैं|
+
+## कण्ट्रोल प्लेन अवयव 
+
+एक बड़े क्लस्टर के लिए, आपको पर्याप्त गणना और अन्य संसाधनों के साथ एक कण्ट्रोल प्लेन की आवश्यकता होती है|
+
+आम तौर पर आप प्रति विफलता क्षेत्र में एक या दो कण्ट्रोल प्लेन उदाहरण चलाएंगे, उन उदाहरणों को लंबवत स्केलिंग और फिर (vertical) स्केलिंग क्षैतिज रूप से स्केलिंग गिरने के बिंदु तक पहुंचने के बाद|
```

Comment:
उदाहरण = Instance. No translation required or we need to find better phrasing for this.